soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -152,6 +152,24 @@ choice SEGGER_SYSVIEW_SECTION
 	depends on SEGGER_SYSTEMVIEW
 endchoice
 
+#
+# MBEDTLS is larger but much faster than TinyCrypt so choose wisely
+#
+config MBEDTLS
+#config TINYCRYPT
+	default y
+	depends on ENTROPY_GENERATOR
```

Comment:
I'm a bit confused about this part, if `XOROSHIRO` or the newer `XOSHIRO` is enabled, why does either of these need to be enabled by default?

Reply:
If the application uses the CSPRNG functions (sys_csrand_get()) it needs an implementation of ctr_drbg. With Zephyr, that either comes from MBEDTLS or TINYCRYPT.

Reply:
Maybe make that explicit?

```
default y if CSPRING_ENABLED
```

Reply:
I like that idea. I'm not fully fluent in Kconfig syntax.